python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -133,6 +133,8 @@ def preprocess(self, payload: Union[Dict, InferRequest], context: Dict[str, Any]
                 max_length=self.max_length,
                 add_special_tokens=self.add_special_tokens,
                 return_tensors=TensorType.NUMPY,
+                padding=True,
```

Comment:
@kevinmingtarja is this always true or we should make them as options

Reply:
I think it's safe to always set to true, i.e. even if we only provide a single input, no padding will be applied, so we don't need to handle that case separately.